config/vufind/config.ini

```
@@ -2158,11 +2158,19 @@ includeSchemaOrgMetadata = true
 [AlphaBrowse]
 ; This setting controls how many headings are displayed on each page of results:
 page_size = 20
+
 ; How many headings to show before the match (or the spot where the match
 ; would have been found). Default is 0 for backwards compatibility.
 rows_before = 0
+
+; For topic browse, controls the separator to place between terms when
+; displaying topic headings.  Defaults to " > " if not set.
+;
+; topic_browse_separator = " > "
```

Small suggested edit to make this match our usual config style (spaces before human-readable comments, no spaces before example settings after comments):

```suggestion
;topic_browse_separator = " > "
```